assets/src/edit-story/components/canvas/displayLayer.js

```diff
@@ -47,10 +47,11 @@ function DisplayLayer() {
 					if ( editingElement === id ) {
 						return null;
 					}
+					const isBackground = currentPage.backgroundElementId === id;
 					return (
 						<DisplayElement
 							key={ id }
-							element={ { id, ...rest } }
+							element={ { id, ...rest, isBackground } }
```

Comment:
This code is quite duplicated over the UI layer. Maybe it'd be better to live with this duplication on the data layer? I think reducers are better equipped to deal with duplication between `page.backgroundElementId` and `element.isBackground`. Alternatively, we could pass `isBackground` everywhere as a separate property as you did for `Save` template. 

So, IMHO, it should be consistent and I do think it might be easier to manage it on data level. Thoughts?

Reply:
Makes sense to be consistent. Moved setting `isBackground` to `setBackgroundElement`.